public/locales/en/gallery.json

```diff
@@ -1 +1,44 @@
-{}
+{
+    "severity": "Severity:",
```

Comment:
Why do you have the translations added here instead of using the translations from the validation interface? I think there are translations for everything except none/not applicable in `validate.json`.

Reply:
yeah, might as well. just for the few that are Gallery-specific. You could make common the default namespace though so you don't have to prepend the common namespace to almost everything

Reply:
done